### DISCUSSION TITLE
Python pour la data science - 3  Comment aborder un jeu de données

### DISCUSSION THREAD

**Discussion by giscus** [linogaliana/python-datascientist#408] [General]
# Python pour la data science - 3  Comment aborder un jeu de données

Quelques éléments pour adopter une démarche scientifique et éthique face à un jeu de données.

https://pythonds.linogaliana.fr/content/getting-started/03_data_analysis.html

**Comment by daniel-odc** [comment]
@lino, le lien "Mise en production de projets data science" ne fonctionne pas

**Reply by linogaliana** [reply]
@daniel-odc c'est bon c'est corrigé, merci ! (887914c).

`@lino` c'est un autre Lino, moi c'est `@linogaliana` 😉
